### ISSUE TITLE
Invalid numbers are reported in wrong location and also prevents all other linting

### ISSUE THREAD

**Issue by PEZ** [clj-kondo/clj-kondo#1058]
**2020.10.10**

**platform**

MacOS
Both the `clj-kondo` executable and the VSCode extension (I think `.jar`?)

**editor**

VSCode the official clj-kondo extension

**problem**

When the text contains an invalid number, such as `0xg` the error is caught, but reported in the wrong location. And then also no other linting is happening.

**repro**

```sh
$ echo "(ns foo)\n0xg\nx" | clj-kondo --lint -
```

**expected behavior**

```
<stdin>:2:1: error: Invalid number: 0xg.
<stdin>:3:1: error: unresolved symbol x
```

**actual behavior**

```
<stdin>:1:3: error: Invalid number: 0xg.
```

**Comment by borkdude** [comment]
It seems this issue was already fixed!

```
$ echo "(ns foo)\n0xg\nx(inc :foo)" | clj-kondo --lint -
<stdin>:2:1: error: Invalid number: 0xg.
<stdin>:3:1: error: Unresolved symbol: x
<stdin>:3:7: warning: Expected: number, received: keyword.
linting took 28ms, errors: 2, warnings: 1
```
